python/setup.py

```diff
@@ -153,21 +153,15 @@ def _supports_symlinks():
     # will search for SPARK_HOME with Python.
     scripts.append("pyspark/find_spark_home.py")
 
-    # Parse the README markdown file into rst for PyPI
-    long_description = "!!!!! missing pandoc do not upload to PyPI !!!!"
-    try:
-        import pypandoc
-        long_description = pypandoc.convert('README.md', 'rst')
-    except ImportError:
-        print("Could not import pypandoc - required to package PySpark", file=sys.stderr)
-    except OSError:
-        print("Could not convert - pandoc is not installed", file=sys.stderr)
+    with open('README.md') as f:
```

Comment:
Yea, I think PyPi renders md files fine without rst.

Reply:
@nchammas, can we remove here too:

https://github.com/apache/spark/blob/04e99c1e1b29b691a8fb51ecfcd7e99482ee0bb3/dev/run-pip-tests#L88

Reply:
We should remove here too https://github.com/apache/spark/blob/master/dev/create-release/spark-rm/Dockerfile#L37

with the comment https://github.com/apache/spark/blob/master/dev/create-release/spark-rm/Dockerfile#L23